app/assets/stylesheets/_mixins.scss

```diff
@@ -4,3 +4,12 @@
   letter-spacing: $letter-spacing-header;
   text-transform: uppercase;
 }
+
+@mixin link-target {
```

Comment:
what's this do, how does it make the empty span something you can (I assume) click on?